pkg/integration/tests/patch_building/toggle_range.go

```diff
@@ -46,10 +43,20 @@ var CheckToggleLogic = NewIntegrationTest(NewIntegrationTestArgs{
 				Contains("  A").Contains("file2-b"),
 				Contains("  A").Contains("file3-b"),
 			).
-			NavigateToLine(Contains("  A").Contains("file1-a")).
+			NavigateToLine(Contains("file1-a")).
 			Press(keys.Universal.ToggleRangeSelect).
 			NavigateToLine(Contains("file3-a")).
 			PressPrimaryAction().
+			Lines(
```

Comment:
Frequently showing what the current state of the app is makes it easier for me to follow the test

Reply:
Agreed, this is much easier to see what's happening.